config/initializers/active_job_logger_patch.rb

```diff
@@ -5,6 +5,13 @@
   module ActiveJob
     module Logging
       class LogSubscriber
+        def enqueue(event)
+          info do
+            job = event.payload[:job]
+            "Enqueued #{job.class.name} (Job ID: #{job.job_id}) to #{queue_name(event)} at: #{Time.zone.now}"
```

Comment:
But also on second thought, what if we used a JSON format? That would make it easier to parse these (and build analytics on job frequency, duration, etc etc)

```
{
  source: "ActiveJob",
  timestamp: "2017-07-12 22:25:32 UTC",
  event: "enqueued",
  job_class: "SmsOtpSenderJob",
  job_queue: "sms",
  job_id: "502339a0-29da-41fb-b5dc-5c44e1b5df75"
}.to_json
```